pkg/update/update_cockroach_version_test.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2021 The Cockroach Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package update
+
+import (
+	"testing"
+
+	semver "github.com/Masterminds/semver/v3"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPerserveMatches(t *testing.T) {
```

Comment:
I need to fix the typos

Reply:
nit: `TestDowngradeAllowed` would be a bit more clear.